sublime_music/ui/player_controls.py

```diff
@@ -52,15 +53,17 @@ def __init__(self):
         Gtk.ActionBar.__init__(self)
         self.set_name("player-controls-bar")
 
+        self.create_rating_buttons()
         song_display = self.create_song_display()
         playback_controls = self.create_playback_controls()
-        play_queue_volume = self.create_play_queue_volume()
+        rating_play_queue_volume = self.create_rating_play_queue_volume()
 
         self.last_device_list_update = None
 
         self.pack_start(song_display)
+        self.pack_start(self.rating_buttons_box)
```

Comment:
I'm confused... isn't this added inside of the `rating_play_queue_volume` component?

Reply:
Indeed. This has now been removed and all the packing happens inside the `rating_play_queue_volume` component